src/components/dashboard/reports/ReportsList.tsx

```diff
@@ -9,20 +9,24 @@ interface ReportsProps {
 export default function ReportsList({ reports, deal }: ReportsProps) {
   return (
     <div className="grid gap-8">
-      {reports.map((report) => (
-        <div
-          key={report.xata_id}
-          className="w-full rounded-md bg-gray-800 px-4 py-8 "
-        >
-          <Link
-            href={`/deals/${deal.xata_id}`}
-            className=" relative text-white hover:text-teal-500"
-            rel="noopener noreferrer"
-          ></Link>
-          <p className="">Reason: {report.reason}</p>
-          <p className="">Email: {report.email}</p>
-        </div>
-      ))}
+      {reports.length > 0 &&
```

Comment:
Instead of `reports.length > 0 &&` and separately calling `reports.length ===0 &&` you could combine it to a turnary `{reports.length > 0 ? <></> : <></>}`